ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -353,7 +353,7 @@ def handleAppletStateUpdateRequested(self):
         batch_processing_busy = self.batchProcessingApplet.busy
 
         self._shell.setAppletEnabled( self.dataSelectionApplet,             not batch_processing_busy )
-        self._shell.setAppletEnabled( self.wsdtApplet,                      not batch_processing_busy and input_ready and not superpixels_available_from_file )
+        self._shell.setAppletEnabled( self.wsdtApplet,                      not batch_processing_busy and input_ready and opDataSelection.ImageGroup[0][1].ready() and not superpixels_available_from_file )
```

Comment:
While your code works for a single lane, it would be nice to have it a bit more generic. Other workflows implement a `_inputReady` method, e.g. in Object Classification:

https://github.com/ilastik/ilastik/blob/428b5180e63f8656b36865d43db0eef62e053314/ilastik/workflows/objectClassification/objectClassificationWorkflow.py#L423-L433

Albeit this is a bit more code, this is more generic and also works for more than a single image lane.

You can find examples of its usage here:

https://github.com/ilastik/ilastik/blob/428b5180e63f8656b36865d43db0eef62e053314/ilastik/workflows/objectClassification/objectClassificationWorkflow.py#L630